modules/custom/d_p_reference_content/d_p_reference_content.module

```diff
@@ -22,6 +22,20 @@ function d_p_reference_content_theme($existing, $type, $theme, $path) {
   ];
 }
 
+/**
+ * Implements hook_preprocess_field().
+ */
+function d_p_reference_content_preprocess_field(&$variables) {
+  if ($variables['element']['#entity_type'] == 'paragraph' && $variables['element']['#bundle'] == 'd_p_reference_content') {
+    if ($variables['field_name'] == 'field_d_long_text') {
+        $variables['attributes']['class'][] = 'container';
+    }
+    if ($variables['field_name'] == 'field_d_media_icon') {
+        $variables['attributes']['class'][] = 'container mx-auto text-center';
+    }
```

Comment:
@tomasz-makiel-droptica a dlaczego w ten sposob, zamiast to w twigu ulozyc i ustawiac klasy?
@grzegorz-pietrzak-droptica co sądzisz?

Reply:
Poprawiłem tak że klasy się w twigu dodają.